packages/checkout-storefront/src/fetch/requests.ts

```diff
@@ -17,82 +17,136 @@ import { PayResult } from "./types";
 import { urlJoinTrailingSlash } from "./urlJoin";
 
 export type PaymentMethodsRequestArgs = {
+  saleorApiUrl: string;
   channelId: string;
   checkoutApiUrl: string;
 };
 
 export const getPaymentMethods = ({
+  saleorApiUrl,
   checkoutApiUrl,
   channelId,
 }: PaymentMethodsRequestArgs): FetchResponse<ChannelActivePaymentProvidersByChannel> =>
-  fetch(urlJoinTrailingSlash(checkoutApiUrl, "active-payment-providers", channelId));
+  fetch(
+    urlJoinTrailingSlash(checkoutApiUrl, "active-payment-providers", channelId) +
+      `?` +
+      new URLSearchParams({ saleorApiUrl }).toString()
```

Comment:
It would be:
```
`?saleorApiUrl=${encodeURIComponent(saleorApiUrl)}`
```
IMO it's less readable and harder to extend if we need to add any other params in the future.